### PR TITLE
TC_01.006.04-A | FreestyleProject > Move project | from Dashboard

### PR DESCRIPTION
**Pull request by hobbita**
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/218

### CODE REVIEW

**Comment by adiuzhikov** [comment]
Please, modify the title as per the instructions and include the link to the TC card in the description.

**Review by Tarantata — APPROVED** [review]

**Review by Infared1 — APPROVED** [review]

File: cypress/e2e/freestyleProjectMoveProject.cy.js [hunk 1]
```diff
@@ -25,6 +25,32 @@ describe ('US_01.006 | FreestyleProject > Move project', () => {
...
+        cy.get('input[id="name"]').click().type('New Job')
+        cy.get('li.hudson_model_FreeStyleProject').click({force: true})
+        cy.get('button').contains('OK').click()
+        cy.get('button').contains('Save').click()
+
+        cy.visit('http://localhost:8080/')
+        cy.get('button.jenkins-menu-dropdown-chevron').eq(3).click({force: true})
+        cy.get('a.jenkins-dropdown__item').contains('Move').invoke('text').then((Text) => {
+            const trimtxt=Text.trim()
+            expect(trimtxt).to.equal('Move')
```
Comment (vukpwnz) [inline cypress/e2e/freestyleProjectMoveProject.cy.js:45]: Hey, sister
you can replace this part of code that checks your Move button contains 'Move' text with little bit shorter assertion.
Try 'cy.get('a.jenkins-dropdown__item').contains(/\s+?Move\s+?/).click()'

**Review by vukpwnz — COMMENTED** [review]
Nice

File: cypress/e2e/freestyleProjectMoveProject.cy.js (hunk 1 above)
Comment (hobbita) [reply cypress/e2e/freestyleProjectMoveProject.cy.js:45]: Nice to see you here =)

**Pull request merged**